widget_scripts.md

```diff
@@ -1,3 +1,112 @@
 
 # Widget scripts
 
+### Widgets (HORUS ONLY) Scripts
+
+##### General description
+
+Widgets are small scripts that show some info in a 'zone' in one of the model specific user defined (telemetry) screens. You can define those screens within the telemetry menu on the HORUS. 
+
+Each model can have up to five custom screens, with up to 8 widgets per screen, depending on their size and layout. Each instance of a widget has his own custom settings.
+
+##### File Location
+
+Widgets are located on the SD card, each in their specific folder /WIDGETS/&lt;*name*&gt;/main.lua (*name* must be in 8 characters or less).
+
```

Comment:
Is 8 chars the limit, not 6? I am not 100% sure here.

Reply:
Actually it can be way more, since it is a directory, and not a registered file within opentx.
8 will definitly work, more I don't know. (movingcounter is already 12 ;))